app/views/helpers/index/tags.phtml

```diff
@@ -0,0 +1,42 @@
+<?php
+	declare(strict_types=1);
+	/** @var FreshRSS_View $this */
+	[$firstTags,$remainingTags] = $this->entry->tagsFormattingHelper();
+?>
+<div class="tags">
+<?php if (!empty($firstTags)): ?>
+	<?= _i('tag') ?><ul class="list-tags">
+	<?php if (Minz_Request::controllerName() === 'index'): ?>
+		<?php foreach ($firstTags as $tag): ?>
+		<li class="item tag"><a class="link-tag" href="<?= _url('index', 'index', 'search', '#' . str_replace(' ', '+', htmlspecialchars_decode($tag, ENT_QUOTES))) ?>" title="<?= _t('gen.action.filter') ?>">#<?= $tag ?></a></li>
+		<?php endforeach; ?>
+	<?php else: // API public access ?>
+		<?php foreach ($firstTags as $tag): ?>
+		<li class="item tag"><a class="link-tag" href="<?= $this->html_url . '&search=%23' . str_replace(' ', '+', htmlspecialchars_decode($tag, ENT_QUOTES)) ?>" title="<?= _t('gen.action.filter') ?>">#<?= $tag ?></a></li>
+		<?php endforeach; ?>
+	<?php endif; ?>
+
+	<?php if (!empty($remainingTags)): // more than 7 tags: show dropdown menu ?>
+		<li class="item tag">
+			<div class="dropdown">
+				<div id="dropdown-tags2-<?= $this->entry->id() ?>" class="dropdown-target"></div>
```

Comment:
This makes an regression :/
If the tags are shown above and below an article the ID needs to be different from eachother (above: `tags` and below `tags2`). 
With `tags.html`, It is always `tags2`. Any idea how to handle it?

ping @Alkarex

Reply:
Quick suggestion: https://github.com/FreshRSS/FreshRSS/pull/6990

Reply:
What is the issue with the number? It is rarely used